src/lib/divider/divider.scss

```diff
@@ -1,26 +1,32 @@
-@use './mixins';
-
-@include mixins.core-styles;
+@use './core' as *;
 
 :host {
-  @include mixins.host;
+  @include host;
 }
 
 :host([hidden]) {
   display: none;
 }
 
-:host-context(forge-drawer) {
-  .forge-divider {
-    margin: var(--forge-divider-drawer-margin, 4px) 0;
-  }
+.forge-divider {
+  @include tokens;
 }
 
+.forge-divider {
+  @include base;
+}
+
+// :host-context(forge-drawer) {
+//   .forge-divider {
+//     margin: var(--forge-divider-drawer-margin, 4px) 0;
+//   }
+// }
```

Comment:
@samrichardsontylertech do you have any opinions on this? I'm tempted to just remove it altogether and let developers styles it themselves. In which case, I think removing the `margin` token/style as mentioned above would be fine since developers can add `margin` themselves to the host.

To replace this we would need to go into the drawer styles and add something along the lines of the following:
```scss
::slotted(forge-divider) {
  @include divider.provide-theme(( margin: 4px ));
}
```

This won't work quite the same as `:host-context` though since it will only targets direct descendants of drawers. We could potentially use `::slotted(*)` but that may have unintended side-effects for deeply nested structures...

Reply:
I think it's reasonable and along the same lines as our other components to let drawers style their dividers. I'm afraid that if we take it out we're just not going to see it implemented consistently. `:host-context` seems like the *correct* way to handle this, but it doesn't have great support. `::slotted(forge-divider)` feels good too but is basically a non-starter because of how easy it is to accidentally circumvent. My gut says to go with `::slotted(*)` and if someone needs to reset the margin they can use the custom property.

That said, I think it makes a lot more sense for `margin` to be set on the host like it would for any other element. So I'd only use the internal declaration for context dependent stuff like this. Or, even better, set the token on the host so it can be easily overridden.

Reply:
Yeah I agree. That's the direction I'm leaning as well.

@jonathanearl-tylertech I'll follow up with you on how we can implement host tokens to allow for setting the margin there instead of internally and we can use `::slotted(*)` for this from the drawer styles.